files/usr/local/www/packages/birdvector/birdvector_about.php

```diff
@@ -0,0 +1,41 @@
+<?php
+require_once("config.inc");
+require_once("guiconfig.inc");
+
+require_once("/usr/local/pkg/birdvector.inc");
+
+$pgtitle = array(gettext("Services"), gettext("Bird Vector"), gettext("About"));
+
+include("head.inc");
+
+$tab_array = array();
+$tab_array[] = array(gettext("Status"), false, "/packages/birdvector/birdvector.php");
+$tab_array[] = array(gettext("Configuration"), true, "/packages/birdvector/birdvector_config.php");
+$tab_array[] = array(gettext("Interactive Shell"), false, "/packages/birdvector/birdvector_shell.php");
+$tab_array[] = array(gettext("About"), false, "/packages/birdvector/birdvector_about.php");
```

Comment:
Need to set the `About` nav to `true` and the rest to `false` to show the correct highlight links